pkgs/applications/audio/famistudio/bin.nix

```
@@ -0,0 +1,65 @@
+{ lib
+, stdenv
+, fetchzip
+, autoPatchelfHook
+, makeWrapper
+, alsa-lib
+, gtk-sharp-2_0
+, glib
+, gtk2
+, mono
+, openal
+}:
+
+stdenv.mkDerivation rec {
+  pname = "famistudio-bin";
```

Is the distinction also made in nixpkgs by the name? I know that firefox has a -bin package, but that's because there's also a from-source one as well as benefits when running the -bin package. 
I think we can just call it famistudio for now

---

I think it's better to make a distinction between from-source and binary-patching from the get-go, when introducing a from-source package later on you avoid the problem of having to change around attribute names and sweeping the rug under some users' legs who want to continue using the official packages.

I don't know about all of Nixpkgs but when a binary-patching package for OpenWatcom 1.9 was introduced back in https://github.com/NixOS/nixpkgs/commit/c26e7f9aac4a12bc206d4461efd228697e1dd3a7 it had the `-bin` suffix too.

---

```suggestion
  pname = "famistudio";
```

We now have https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/meta.chapter.md#source-provenance-sec-meta-sourceprovenance to properly do this.